cfme/tests/containers/test_containers_projects_summary.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.project import Project
+from cfme.web_ui import CheckboxTable
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+
+projects_properties_fields = ['Name', 'Creation timestamp', 'Resource version']
+projects_relationships_fields = ['Containers Provider', 'Routes', 'Services', 'Replicators',
+                                 'Pods', 'Nodes']
+
+list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+
+
+# CMP - 9523
```

Comment:
is this one case in Polarion or two? if its two, please add the other TC number
also I think that you can use parametarization here instead of code duplication
its the same test just checking a different field name